Generative-AI/Ai_voice_agent/app/main.py

```
@@ -0,0 +1,40 @@
+async def root():
+    # Serve your frontend index.html here from static folder explicitly
+    index_path = os.path.join("static", "index.html")
+    return HTMLResponse(content=open(index_path).read(), status_code=200)
+
+@app.post("/inbound-call")
+async def inbound_call(data: dict):
+    try:
+        result = await handle_inbound_call(data)
+        return JSONResponse(content=result)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)